### PR TITLE
fix(coachindicator): adds 'static white', deprecates light/dark

### PR DESCRIPTION
**Pull request by 5t3ph**
## Description

Per [feedback from PJ](https://adobedesign.slack.com/archives/GEN034MAS/p1729092256236909) on expected variants for Coach indicator, marks "light" and "dark" as deprecated for S2, and adds "static white".

[According to Spectrum stats](https://awesome.adproto.com/spectrum-metric-web-app-main/#/@spectrum-web-components/components), it seems this component is very low usage overall and also that the default variant is the only one in use in products, if at all.

## How and where has this been tested?

Locally in Storybook.

### Validation steps

In Storybook, find the "Coach indicator" docs page to review the added "Static white" story.

[PR deploy](https://pr-3266--spectrum-css.netlify.app/preview/?path=/docs/components-coach-indicator--docs)

### Regression testing

Validate:

1. The documentation pages for at least two other components are still loading, including:

- [x] The pages render correctly, are accessible, and are responsive.

2. If components have been modified, VRTs have been run on this branch:

- [ ] VRTs have been run and looked at.
- [ ] Any VRT changes have been accepted (by reviewer and/or PR author), or there are no changes.

## Screenshots

image

## To-do list

- [x] I have read the [contribution guidelines](/.github/CONTRIBUTING.md).
- [x] I have updated relevant storybook stories and templates.
- [x] I have tested these changes in Windows High Contrast mode.
- [x] If my change impacts **other components**, I have tested to make sure they don't break.
- [x] If my change impacts **documentation**, I have updated the documentation accordingly.
- [ ] ✨ This pull request is ready to merge. ✨

### CODE REVIEW

**Review by rise-erpelding — CHANGES_REQUESTED** [review]
The CSS for coach indicator and tokens looks great! I made a couple of small suggestions for the Storybook side of things to better align with how we've done other components.

File: tokens/custom-spectrum/custom-vars.css [hunk 1]
```diff
@@ -53,6 +53,8 @@
   --spectrum-cjk-font: var(--spectrum-code-font-family-stack);
 
   /* static white / black background color for docs containers */
-  --spectrum-docs-static-white-background-color: rgb(15, 121, 125);
-  --spectrum-docs-static-black-background-color: rgb(181, 209, 211);
+	--spectrum-docs-static-white-background-color: rgb(15, 121, 125);
```
Comment (5t3ph) [inline tokens/custom-spectrum/custom-vars.css:56]: I'm not sure why this indentation is happening on these token files, it's not like that locally :(

File: components/coachindicator/stories/coachindicator.stories.js [hunk 2]
```diff
@@ -22,7 +24,7 @@ export default {
 				type: { summary: "string" },
 				category: "Component",
 			},
-			options: ["default", "dark", "light"],
+			options: ["default", "dark", "light", "staticWhite"],
```
[the remark this answers is not in the document]
Comment (5t3ph) [reply components/coachindicator/stories/coachindicator.stories.js]: @rise-erpelding thanks for sharing, I've updated as you suggested!

**Review by rise-erpelding — APPROVED** [review]
Looks great!

**Review by cdransf — APPROVED** [review]

**Pull request merged**